tests/unittests/media/client/ipc/webAudioPlayerIpc/CreateTest.cpp

```diff
@@ -24,7 +24,13 @@ MATCHER_P3(createWebAudioPlayerRequestMatcher, audioMimeType, priority, config,
 {
     const ::firebolt::rialto::CreateWebAudioPlayerRequest *kRequest =
         dynamic_cast<const ::firebolt::rialto::CreateWebAudioPlayerRequest *>(arg);
-    return ((kRequest->audio_mime_type() == audioMimeType) && (kRequest->priority() == priority));
```

Comment:
Use the matcher in common/matchers/WebAudioPlayerProtoRequests.h. Can deliver this and make the change in your next PR.